python/ray/data/dataset.py

```diff
@@ -1971,65 +1968,21 @@ def iter_batches(
             drop_last: Whether to drop the last batch if it's incomplete.
 
         Returns:
-            A list of iterators over record batches.
+            An iterator over record batches.
```

Comment:
Not sure why the return description was "A list of iterators over record batches" prior to this, since it should just be an iterator over record batches that's returned.